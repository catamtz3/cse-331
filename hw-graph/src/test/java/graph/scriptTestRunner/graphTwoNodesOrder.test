# Create a graph
CreateGraph sortedGraph

#Add two nodes to the graph
AddNode sortedGraph b
AddNode sortedGraph a

# Get the nodes of the graph
ListNodes sortedGraph