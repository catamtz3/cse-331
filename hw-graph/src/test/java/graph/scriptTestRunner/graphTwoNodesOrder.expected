# Create a graph
Created graph sortedGraph

#Add two nodes to the graph
Added node b to sortedGraph
Added node a to sortedGraph


# Get the nodes of the graph
sortedGraph contains: a, b